### PR TITLE
Add a minor caveat about node uptimes

### PR DESCRIPTION
**Pull request by astorije**
**Description:**

**Review:**

- [ ] Do the unit tests contain positive functionality tests as well as edge case negative tests?
- [ ] Can we monitor the performance/effectiveness of these changes?
- [ ] Did the code review consider failure modes and edge cases?
- [ ] Did the code review consider refactorings that would make this code more maintainable?
- [ ] Was this change approved by a module owner (if necessary)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ installed via `apt-get` (`sudo apt-get install python-httplib2`) or via `pip`
 (`sudo pip install httplib2`) if not available in the distribution package
 manager.
 
+Additionally, there is a current limitation regarding the `uptime` of your
+nodes: you need to either use Ansible 2+, or to have `ohai` or `facter`
+installed on your managed nodes. Failing to do so will simply leave your
```
Comment (aschepis) [inline README.md]: remove `simply`

**Comment by aschepis** [comment]
:+1: after one fix.
